roles/glance/templates/etc/glance/glance-swift-store.conf

```diff
@@ -0,0 +1,11 @@
+[ref1]
+{% if (glance.store_smart|bool and swift.enabled|bool) or
+      glance.store_swift|bool  %}
```

Comment:
I don't see why it matters. There are plenty of passwords written to the filesystem already. It just seems redundant to have extra conditionals here. In fact, this password is already written to /etc/glance.conf.